README.md

```diff
@@ -1,5 +1,8 @@
+
 # 🍍 durian
 
+[![codecov](https://codecov.io/gh/ubclaunchpad/durian/branch/master/graph/badge.svg)](https://codecov.io/gh/ubclaunchpad/durian)
```

Comment:
while you're at it, can you chuck the travis badge here as well?

Reply:
Can do.